featuretools/tests/primitive_tests/test_feature_base.py

```diff
@@ -208,3 +209,40 @@ def test_multi_output_index_error(es):
     error_text = 'index is higher than the number of outputs'
     with pytest.raises(AssertionError, match=error_text):
         three_common[10]
+
+
+def test_rename(es):
+    feat = ft.Feature(es['log']['id'], parent_entity=es['sessions'], primitive=Count)
+    feat.get_feature_names()
+    copy_feat = feat.rename("session_test")
+    assert feat.unique_name() != copy_feat.unique_name()
```

Comment:
for these checks, can we also check what we expect the name to be, along with the confirming the name changed?

can we turn these checks into a reusable function since all three tests share the same checks

Reply:
Yes